### DISCUSSION TITLE
Responsive Menu on "small" screens

### DISCUSSION THREAD

**Discussion by rainerrose** [razonyang/hugo-theme-bootstrap#1089] [Q&A] [answered]
Hi!

The responsive design works almost perfect. On some “smaller” screens, IMHO the menu is reduced too early.

## Example

Let's take the following website [https://hbs.razonyang.com/](https://hbs.razonyang.com/ ).

## more than 1400px 

A screen with more than 1400px the menu is displayed in full width at the top.

![image](https://github.com/user-attachments/assets/4647fb8f-4253-4cca-8f00-682fe278369a)

## Between 1300px and 1399px 

Now reduce the resolution (like my laptop with 1360px width).
All the icons vanished and I only see a very reduced view of the menu; see screenshot

![image](https://github.com/user-attachments/assets/3959b8af-6256-44cb-8695-60b240c6e518)

## Reduce Zoom-Factor

But when I reduce the zoom-factor in my browser to 90% you got this:

![90percent](https://github.com/user-attachments/assets/af4cd27d-752c-40c2-81dd-4a1513162984)

IMHO, there (In the screenshot see number 2) should be enough space, even in the 100% variant, so that no reduction should be necessary at the top of the menu.

Is there a way to control this behavior, when the “smaller” menu becomes active?

**Comment by razonyang** [comment, marked as answer]
> Is there a way to control this behavior, when the “smaller” menu becomes active?

Yes, the simplest way is to change the breakpoint size, see https://github.com/razonyang/hugo-theme-bootstrap/discussions/1082#discussioncomment-11284541

**Reply by rainerrose** [reply]
Thanks a lot. I did search in the discussions, but obviously not well enough. Sorry.

**Reply by razonyang** [reply]
No problem.

I just added a parameter called `topAppBar.expandBreakpoint` (`xl`, `lg`, `xxl`, default to `xxl`), available since `v1.9.0`, you can now set it on params file to achieve this.
